web/hasura/metadata/tables.yaml

```diff
@@ -100,6 +100,9 @@
   - name: bountyStatusType
     using:
       foreign_key_constraint_on: status
+  - name: experienceLevelByExperiencelevel
```

Comment:
what's that?

Reply:
I didn't touch this file

Reply:
well, there is a change. changes to this files are done through the hasura console (`y db:console`). 
Revert this file and then run `y db:create`